documentation/reference/search-definitions-reference.html

```diff
@@ -2554,11 +2554,11 @@ <h2 id="field-types">Field types</h2>
 <ul>
   <li>Struct fields can <strong>only</strong> be searched in
       <a href="../streaming-search.html">streaming search</a> mode,
-      <strong>not</strong> in indexed search</li>
+      <strong>not</strong> in indexed search (but <em>arrays of struct</em> are searchable, see below)</li>
   <li>Struct fields can be fed, retrieved and used in document summaries in both indexed and streaming search</li>
 </ul>
-See <a href="#type:array">struct array type</a> and
-<a href="#type:map">map type</a> for restrictions when using collections of structs.
+Search in <a href="#type:array">array&lt;struct&gt;</a> is supported.
```

Comment:
There are also restrictions for when you can search array of structs, similar to map of structs. This is described in `#type:array.` Consider reformulate.